authorization-service/handlers/basicAuthorizer.handler.ts

```diff
@@ -0,0 +1,31 @@
+import { APIGatewayTokenAuthorizerHandler, APIGatewayTokenAuthorizerEvent, Context, APIGatewayAuthorizerResult } from 'aws-lambda';
+import { generatePolicy } from './authorization.helper';
+
+export const basicAuthorizer: APIGatewayTokenAuthorizerHandler = async (event: APIGatewayTokenAuthorizerEvent, _context: Context): Promise<APIGatewayAuthorizerResult> => {
+    // console.log('CONTEXT: [%o]', _context)
+    console.log('EVENT: [%o]', event)
+
+    return new Promise((resolve, reject) => {
```

Comment:
Mixing async / await and Promise looks a little overkill.

Reply:
Yes. Thanks. Fixed.